backend/common/rhnConfig.py

```diff
@@ -539,7 +539,7 @@ def initCFG(component=None, root=None, filename=None):
     ALL_CFG = RHNOptions('')
     ALL_CFG.parse()
     PRODUCT_NAME = ALL_CFG.PRODUCT_NAME
-except ConfigParserError:
+except (ConfigParserError, AttributeError):
```

Comment:
Not sure if this is 100% correct, did the exception change? Now it seems we get an `AttributeError` when the product is SUSE Manager... or maybe this is related to the change above when reading the configfile as text instead of binary.

Reply:
Rolled back as well, it seems the AttributeError was a correct exception, as `.parse` did not parse anything.